src/User32.Desktop/User32.cs

```diff
@@ -621,10 +674,478 @@ public static partial class User32
+        /// </summary>
+        /// <param name="lpszDesktop">The name of the desktop to be created. Desktop names are case-insensitive and may not contain backslash characters (\).</param>
+        /// <param name="lpszDevice">This parameter is reserved and must be <see cref="IntPtr.Zero"/></param>
+        /// <param name="pDevmode">This parameter is reserved and must be <see cref="IntPtr.Zero"/>.</param>
+        /// <param name="dwFlags">Access control flags</param>
+        /// <param name="dwDesiredAccess">
+        /// The requested access to the desktop. For a list of values, see <see cref="DESKTOP_ACCESS_MASK"/>.
+        /// This parameter must include the <see cref="DESKTOP_ACCESS_MASK.SpecificRight.DESKTOP_CREATEWINDOW"/> access right, because internally <see cref="CreateDesktop(string, string, IntPtr, DesktopCreationFlags, DESKTOP_ACCESS_MASK, Kernel32.SECURITY_ATTRIBUTES*)"/> uses the handle to create a window.
+        /// </param>
+        /// <param name="lpsa">
+        /// A pointer to a <see cref="Kernel32.SECURITY_ATTRIBUTES"/> structure that determines whether the returned handle can be inherited by child processes. If lpsa is NULL, the handle cannot be inherited.
+        /// The <see cref="Kernel32.SECURITY_ATTRIBUTES.lpSecurityDescriptor"/> member of the structure specifies a security descriptor for the new desktop. If this parameter is NULL, the desktop inherits its security descriptor from the parent window station.
+        /// </param>
+        /// <returns>
+        /// If the function succeeds, the return value is a handle to the newly created desktop.
+        /// If the specified desktop already exists, the function succeeds and returns a handle to the existing desktop.
+        /// If the function fails, the return value is an invalid handle.
+        /// </returns>
+        /// <remarks>
+        /// <para>
+        /// If the dwDesiredAccess parameter specifies the READ_CONTROL, WRITE_DAC, or WRITE_OWNER standard access rights, you must also request the DESKTOP_READOBJECTS and DESKTOP_WRITEOBJECTS access rights.
+        /// </para>
+        /// <para>
+        /// The number of desktops that can be created is limited by the size of the system desktop heap, which is 48 MB. Desktop objects use the heap to store resources. You can increase the number of desktops that can be created by reducing the default heap reserved for each desktop in the interactive window station. This value is specified in the SharedSection substring of the following registry value: HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Session Manager\SubSystems\Windows.
+        /// </para>
+        /// </remarks>
+        [DllImport(nameof(User32), CharSet = CharSet.Unicode, ExactSpelling = true, SetLastError = true)]
```

Comment:
Your use of `ExactSpelling=true` looks wrong. In fact the exported function is `CreateDesktopW` for Unicode. I think using `ExactSpelling=true` makes .NET not find the one with the W suffix.